service/docs/source/geopmaccess.1.rst

```diff
@@ -1,28 +1,181 @@
+user through the service.  The purpose of the ``geopmaccess`` command
+line tool is to enable reading and writing of these access list files.
+
+
+Options
+~~~~~~~
+
+*
+  ``-c``, ``--controls``:
+  Command applies to controls not signals.
+
+
+* ``-u``, ``--user``:
+  Read the default user access list.
+
+*
+  ``-g``, ``--group``:
+  Read or write the access list for a specific Unix GROUP.
+
+*
+  ``-a``, ``--all``:
+  Print all signals or controls supported by the service system (not
+  valid with ``-w`` option).
+
+*
+  ``-w``, ``--write``:
+  Write default access list, or the access list for a particular Unix
+  group.
+
+*
+  ``-D``, ``--delete``:
+  Remove an access list for default user or a particular Unix Group.
+*
+  ``-n``, ``--dry-run``:
+  Do error checking on all user input, but do not modify configuration
+  files
+
+*
+  ``-E``, ``--skip-check``:
+  Write access list to disk without error checking.
+
+*
+  ``-h``, ``--help``:
+  Print brief summary of the command line usage information,
+  then exit.
+
+*
+  ``-v``, ``--version``:
+  Print version of `geopm(7) <geopm.7.html>`_ to standard output, then exit.
+
+
+
+Query Access
+~~~~~~~~~~~~
+
+The default behavior of ``geopmaccess`` when no options are provided
+is to print the list of signals that the calling user has permissions
+to read through the GEOPM Service. This command is used by a GEOPM
+service client to discover which signals they have permission to read
+based on the configuration set by the system administrator.  All
+supported signals or controls are printed when called by a root user.
+
+All ``geopmaccess`` commands refer to reading or writing signal access
+lists by default.  To specify that the command should instead refer to
+the access lists for controls, the ``-c`` / ``--controls`` option must
+be provided.  For example, when the ``-c`` option is provided without
+any other options, the list of controls that may be configured by the
+calling user is printed.
+
+All users of the system will have access to the signals and controls
+determined by the "default access list."  The default access list may
+be read by specifying the ``-u`` / ``--user`` option.  These
+permissions are extended for each user based on Unix user group
+membership.  Each Unix group on the system may have GEOPM Service
+signal and control access lists associated with it.  To read or write
+the access list for a particular Unix user group, the ``-g`` or
+``--group`` option must be specified.
+
+
+Access Management
+~~~~~~~~~~~~~~~~~
+
+Reading access lists may be done by an unprivileged client or by the
+system administrator, but only a process with the Linux capability
+``CAP_SYS_ADMIN`` has permission to write or delete an access list.
+The administrator may execute ``geopmaccess`` to write to an access
+list by providing the ``-w`` / ``--write`` command line option.  The
+``-D`` / ``--delete`` option will remove all signals or controls from
+the configuration.
```

Comment:
The default behavior is different in reads vs writes. In reads, it defaults to the signals of the calling process. In writes/deletes it defaults to the user-default list. Maybe make the difference stand out by ending this paragraph with the reading default behavior, to mirror the end of the writing paragraph.